src/board.py

```diff
@@ -0,0 +1,20 @@
+class Board:
+    border = "⊱ –––––– {⋆⌘⋆} –––––– ⊰",
+    divider = "      ---+---+---"
+
+    def __init__(self):
+        self.spaces = [" ", " ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
This is nice and simple!
Why are you choosing empty strings as places?
Does the board need to know what a string is?
Seems like a concern for how the board gets displayed more than how the board is stored in memory.

Reply:
I chose empty strings as placeholders because I think that looks much cleaner, I've seen people use numbering and things like that but to me, it looks clunky. It may be something we need to talk to the client about though...

I'm not sure what you mean by this:
>Does the board need to know what a string is?
Do you mean the divider? I suppose I can and maybe should have the cli store that instead, thats something we may need to discuss for a couple minutes.